src/components/activity-groups-list/activity-group-item/activity-item/unknown.tsx

```diff
@@ -25,7 +25,7 @@ import { ActivityGroupItemSelectors } from '../selectors';
 import { AbstractItem } from './abstract-item';
 import { ActivityItemProps } from './item.props';
 
-export const Unknown: FC<ActivityItemProps> = ({ activity }) => {
+export const Unknown: FC<ActivityItemProps> = memo(({ activity }) => {
```

Comment:
Please consider using `isEqual` comparison function for all new memo components that have an object as at least one of their properties